hastexo/public/js/main.js

```diff
@@ -270,6 +273,52 @@ function HastexoXBlock(runtime, element, configuration) {
         });
     };
 
+    var construct_layout = function() {
+        var instructions_layout = configuration.instructions_layout;
+
+        /* 'above' is the default layout and doesn't require any changes */
+        if (instructions_layout != 'above') {
+            var content = document.getElementsByClassName("vert-mod")[0];
```

Comment:
I *think* you can make this a little more readable by using a JQuery selector, like we do for selecting `dialog_container`.